src/social.firebase-facebook.json

```diff
@@ -9,13 +9,141 @@
       "facebook-social-provider.js"
    	]
   },
-  "default": "social",
+  "default": "social2",
   "provides": [
-    "social"
+    "social2"
   ],
+  "api": {
+    "social2": {
+      "ERRCODE": {"type": "constant", "value": {
+        "SUCCESS": "Success!",
+        "UNKNOWN": "Unknown error",
+        "OFFLINE": "User is currently offline",
+        "MALFORMEDPARAMETERS": "Parameters are malformed",
+        "LOGIN_BADCREDENTIALS": "Error authenticating with server",
+        "LOGIN_FAILEDCONNECTION": "Error connecting to server",
+        "LOGIN_ALREADYONLINE": "User is already logged in",
+        "LOGIN_OAUTHERROR": "OAuth Error",
+        "SEND_INVALIDDESTINATION": "Message sent to an invalid destination"
+      }},
+
+      "STATUS": {"type": "constant", "value": {
+        "OFFLINE": "OFFLINE",
+        "ONLINE": "ONLINE",
+        "ONLINE_WITH_OTHER_APP": "ONLINE_WITH_OTHER_APP"
+      }},
+
+      "login": {
+        "type": "method",
+        "value": [{
+          "agent": "string",
+          "version": "string",
+          "url": "string",
+          "interactive": "boolean",
+          "rememberLogin": "boolean"
+        }],
+        "ret": {
+          "userId": "string",
+          "clientId": "string",
+          "status": "string",
+          "lastUpdated": "number",
+          "lastSeen": "number"
+        },
+        "err": {
+          "errcode": "string",
+          "message": "string"
+        }
+      },
+
+      "inviteUser": {
+        "type": "method",
+        "value": ["string"],
+        "ret": "object"
+      },
+
+      "acceptUserInvitation": {
+       "type": "method",
+        "value": ["string"]
+      },
+
+      "sendEmail": {
+        "type": "method",
+        "value": ["string", "string", "string"]
+      },
+
+      "clearCachedCredentials": {"type": "method", "value": []},
```

Comment:
optional nit: make the formatting consistent for the methods (e.g. "type", "value" on new lines)
same for onMessage, onUserProfile, onClientState below, and for social.firebase-google.json